scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -524,7 +524,11 @@ def _get_containers(client, ignore_container=None, restrict_to_container=None, l
                                             logger.log( scalyr_logging.DEBUG_LEVEL_2, "Including container '%s' based on pod annotations, %s" % (short_cid, str(pod.annotations)) )
 
                         except Exception, e:
-                          logger.error("Error inspecting container '%s' - %s, %s" % (cid, e, traceback.format_exc()), limit_once_per_x_secs=300,limit_key="docker-api-inspect")
+                          logger.exception(
+                              "Error inspecting container '%s' - %s, %s" % (cid, e, traceback.format_exc()),
```

Comment:
Be sure to put `str(e)` to serialize it to a string.

Reply:
See above about `%` arguments.  I don't think this is needed.